includes/Checker/Checks.php

```diff
@@ -25,47 +25,25 @@ class Checks {
 	protected $checks;
 
 	/**
-	 * Context for the plugin to check.
+	 * Constructor.
 	 *
 	 * @since n.e.x.t
-	 * @var Check_Context
 	 */
-	protected $check_context;
-
-	/**
-	 * Sets the main context and the main file of the plugin to check.
-	 *
-	 * @since n.e.x.t
-	 *
-	 * @param string $plugin_main_file Absolute path to the plugin main file.
-	 */
-	public function __construct( $plugin_main_file ) {
-		$this->check_context = new Check_Context( $plugin_main_file );
-	}
-
-	/**
-	 * Returns the Check Context.
-	 *
-	 * @since n.e.x.t
-	 *
-	 * @return Check_Context The plugin context that is being checked.
-	 */
-	public function context() {
-		return $this->check_context;
-	}
+	public function __construct() { }
```

Comment:
This line can be removed.

Reply:
Thanks @spacedmonkey, Address in https://github.com/10up/plugin-check/pull/167/commits/efa0818f9985e2f8d1fde088d1ec576f63fc24be